server/chat-plugins/quotes.ts

```diff
@@ -44,6 +44,23 @@ export const commands: ChatCommands = {
 		return this.sendReplyBox(`${Chat.getReadmoreBlock(quote)}${attribution}`);
 	},
 	randquotehelp: [`/randquote [showauthor] - Show a random quote from the room. Add 'showauthor' to see who added it and when.`],
+	
+	nquote(target, room, user) {
+		room = this.requireRoom();
+		const roomQuotes = quotes[room.roomid];
+		const index = parseInt(target.trim());
+		if (isNaN(index)) {
+			return this.errorReply(`Invalid index.`);
+		}
+		if(!roomQuotes[index - 1]) {
+			return this.errorReply(`Out of bounds.`);
+		}
+		this.runBroadcast(true);
```

Comment:
Why bypass the broadcast timer? No need to show this more than once, no?

Reply:
```suggestion
		this.runBroadcast();
```
(This is how you avoid bypassing the broadcast timer.)